src/geo/ui/legends/custom/legend-view.js

```diff
@@ -3,9 +3,21 @@ var template = require('./legend-template.tpl');
 
 var CustomLegendView = LegendViewBase.extend({
   _getCompiledTemplate: function () {
-    return template({
-      items: this.model.get('items')
-    });
+    var htmlTemplate = this.model.get('html');
+    if (htmlTemplate === '') {
+      htmlTemplate = template({
+        items: this.model.get('items')
+      });
+    } else {
+      htmlTemplate = this._patchStylesForIE(htmlTemplate);
+    }
+    return htmlTemplate;
+  },
+
+  _patchStylesForIE: function (str) {
```

Comment:
We need to insert a css rule in the inline style declaration to make the legend visible for IE11, non sense, I know. IE11 doesn't show the bullet's color if only that style is declared 😞 .